pkgs/development/python-modules/uamqp/default.nix

```
@@ -23,6 +23,10 @@ buildPythonPackage rec {
     sha256 = "sha256-guhfOMvddC4E+oOmvpeG8GsXEfqLcSHVdtj3w8fF2Vs=";
   };
 
+  patches = lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
```

Why the `isx86_64` check here? It doesn't seem necessary.

---

Because aarch64 fails _with_ it.

---

`aarch64-darwin` is failing anyway it seems. Is this expected?

---

```suggestion
  patches = lib.optionals stdenv.isx86_64 [
```

how about? Maybe we can update the patch like this the next time instead of removing it.